test/system/710-kube.bats

```diff
@@ -74,6 +74,22 @@ status                           | =  | null
     run_podman rm $cname
 }
 
+@test "podman kube generate unmasked" {
+      KUBE=$PODMAN_TMPDIR/kube.yaml
+      run_podman create --name test --security-opt unmask=all $IMAGE
+      run_podman inspect --format '{{ .HostConfig.SecurityOpt }}' test
+      is "$output" "[unmask=all]" "Inspect should see unmask all"
+      run_podman kube generate test -f $KUBE
+      run grep procMount $KUBE
```

Comment:
Since we don't check `$status`, this could fail.

A more bash-idiomatic way would be `assert "$(< $KUBE)" =~ "procMount" "..."`

I would also check for the value of the `procMount` field.